Modules/Common/include/Common/BigScreenCanvas.h

```diff
@@ -0,0 +1,64 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file    BigScreenCanvas.h
+/// \author  Andrea Ferrero
+/// \brief   Canvas showing the aggregated quality of configurable groups
+///
+
+#ifndef QUALITYCONTROL_BIGSCREENCANVAS_H
+#define QUALITYCONTROL_BIGSCREENCANVAS_H
+
+#include "QualityControl/Quality.h"
+#include <TCanvas.h>
+#include <string>
+#include <unordered_map>
+
+namespace o2::quality_control_modules::common
+{
+
+struct BigScreenElement;
+
+class BigScreenCanvas : public TCanvas
+{
+ public:
+  BigScreenCanvas(std::string name, std::string title, int nRows, int nCols, int borderWidth);
+  ~BigScreenCanvas() = default;
+
+  /// \brief add a box in the canvas at a given index, with "name" displayed above the box
+  void add(std::string name, int index);
```

Comment:
```suggestion
  void addBox(std::string name, int index);
```
How about addBox then?

Also, could you document how are the boxes indexed? I.e. does the index start at 0 or 1, does it go vertically or horizontally first?

Reply:
Done